src/main/kotlin/me/partlysanestudios/partlysaneskies/features/mining/events/MiningEvent.kt

```diff
@@ -0,0 +1,132 @@
+package me.partlysanestudios.partlysaneskies.features.mining.events
```

Comment:
Other solution would be to make individual pss events for each of these

Reply:
Either that or create a DwarvenMinesEvent() that has a property of the type

Reply:
what